functions.php

```diff
@@ -261,7 +261,19 @@ function getTotalCol($question,$project_id,$question_1,$conditionDate,$topScoreM
         }
     }
 
-    $missingOverall += $missing_col;
+    $row_questions_1 = ProjectData::getRowQuestionsParticipantPerception();
+    $data = implode(",",$row_questions_1);
+    $RecordSetOverallMissing = \REDCap::getData($project_id, 'array', $recordIds, array($data), null, null, false, false, false, "[".$question_1."] = ''".$conditionDate);
```

Comment:
I don't believe this works as expected, which means this change was not fully tested before committing.  Please fully test all change before committing going forward.  If you're not sure how to fully test something, please ask.

Reply:
I made a test but I think it was too small to realize that the data was working but not giving the right numbers 🤦‍♀️
Now I'm getting the correct data. I made a bigger sized test.

Reply:
Thank you for checking this! I'll take a note to use the bigger sample set as reference.

Reply:
In my experience smaller test cases that more closely target the specific code changes made (similar to unit tests) are actually more effective at verifying changes like this than bigger sample sets that may or may not actually exercise the code that was changed.  They'll also typically quicker.  I wrote a quick test for this line in under a minute!

Reply:
👍